deploy/kubernetes/overlays/stable/kustomization.yaml

```diff
@@ -1,16 +1,15 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 bases:
-- ../../base
+  - ../../base
 images:
-- name: amazon/aws-ebs-csi-driver
-  newTag: v0.7.1
-- name: quay.io/k8scsi/csi-provisioner
-  newTag: v1.5.0
-- name: quay.io/k8scsi/csi-attacher
-  newTag: v1.2.0
-- name: quay.io/k8scsi/livenessprobe
-  newTag: v1.1.0
-- name: quay.io/k8scsi/csi-node-driver-registrar
-  newTag: v1.1.0
-
+  - name: k8s.gcr.io/provider-aws/aws-ebs-csi-driver
+    newTag: v0.8.0
+  - name: quay.io/k8scsi/csi-provisioner
```

Comment:
Still supposed to be quay?

Reply:
yes, sidecars outdated and quay but is separate issue